datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/SchemaTab.tsx

```diff
@@ -49,6 +57,57 @@ export const SchemaTab = ({ properties }: { properties?: any }) => {
     );
 
     const [showKeySchema, setShowKeySchema] = useState(false);
+    const [schemaBlameView, setSchemaBlameView] = useState(SchemaBlameViewType.NORMAL);
```

Comment:
This is really more like

schemaViewMode. 

The name of the variable really threw me off

Reply:
good point, changed